website/content/docs/examples/bootstrap-k0s.md

```diff
@@ -27,10 +27,16 @@ cluster_name = "k0s-cluster"
 controller_count = 1
 worker_count = 1
 cluster_flavor = "m5.large"
+region = "us-east-1"
 ```
 2. `terraform init`
 3. `terraform apply -auto-approve`
 4. `terraform output --raw k0s_cluster > VMs.yaml`
+5. Confirm that the AWS EC2 VMs are spun up:
```

Comment:
Thinking about this more, the previous command returns a list of the VMs that were created and verifies they were spun up. Instead of making 5 and 6 required steps, can I have you change them to a note that is more of an FYI for the user?

Something like this
> Details about the created VMs can be viewed using 
```
aws ec2 describe-instances --region $(grep "region" terraform.tfvars | awk -F' *= *' '{print $2}' | tr -d '"')
```
or by navigating to the AWS EC2 page and selecting the appropriate region from the dropdown in the top right of the page.